src/PKSim.Core/Services/IndividualExtractor.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Formulas;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility.Extensions;
+using PKSim.Assets;
+using PKSim.Core.Model;
+using PKSim.Core.Repositories;
+
+namespace PKSim.Core.Services
+{
+   public interface IIndividualExtractor
+   {
+      void ExtractIndividualsFrom(Population population, params int[] individualIds);
```

Comment:
This is where the extraction is happening: From a given population and a bunch of individualIds, create one new individual for each